OMGVA PoS/Data layer/Models/Payment.cs

```diff
@@ -4,9 +4,18 @@ namespace OMGVA_PoS.Data_layer.Models
 {
     public class Payment
     {
-        public string Id { get; set; }
+        public long Id { get; set; }
```

Comment:
About this one. In the specification it was mentioned that it is specifically a string. I believe it is because stripe uses strings for ids. If I am correct (I do not know much about stripe), then it should be string.

Reply:
Fair enough, reverted, changed PaymentId in Order.cs to string as well